waku/v2/node/config.nim

```diff
@@ -265,6 +265,24 @@ type
       defaultValue: false
       name: "discv5-enr-auto-update" .}: bool
 
+    discv5TableIpLimit* {.
+      hidden
+      desc: "Maximum amount of nodes with the same IP in discv5 routing tables"
+      defaultValue: 10
+      name: "discv5-table-ip-limit" .}: uint
+
+    discv5BucketIpLimit* {.
+      hidden
+      desc: "Maximum amount of nodes with the same IP in discv5 routing table buckets"
+      defaultValue: 2
+      name: "discv5-bucket-ip-limit" .}: uint
+
+    discv5BitsPerHop* {.
+      hidden
+      desc: "Kademlia's b variable, increase for less hops per lookup"
+      defaultValue: 1
+      name: "discv5-bits-per-hop" .}: int
+
```

Comment:
I'm not convinced that these need to be exposed as user-configurable items, at least for now with our very basic requirements. Unless we can define a clear use case where "normal" operators would need to fine tune this, it may be prudent to set these defaults inside the code for now. As we encounter more and more exotic use cases we can expose config items as necessary. Wdyt?

Reply:
Update: Reviewing https://github.com/status-im/nim-waku/pull/863 I notice that at least some of these items are useful when setting up local tests, which I think is a good enough reason for their existence.

Reply:
At some point we might want to factor out admin/dev options vs things users are likely to want to specify. For example with a "namespace"